### DISCUSSION TITLE
When is the next release ?

### DISCUSSION THREAD

**Discussion by ohbibi-bhallion** [arthurfiorette/axios-cache-interceptor#667] [General]
Hi !
the plugin was not released for quite some time, any idea when the next release will land ?

**Comment by arthurfiorette** [comment]
Hey @bhallionOhbibi, I had forgotten to release it.

https://www.npmjs.com/package/axios-cache-interceptor/v/1.3.0

**Reply by ohbibi-bhallion** [reply]
Thanks !!
